Greeter feedback and improvements

capsia37 opened the pull request:
- Darken the background instead of blurring the login area to prevent flashing rectangle when unlocking
- Don't show infographics if space is not enough
- Show infographics on cover for single user tablets

capsia37 committed:
Replace blur on greeter with darkening

Darken the background instead of blurring the login area to prevent flashing rectangle when unlocking

peat-psuwit commented:
Is it possible for you to record a video showing the effect before & after this change? Some hint for screencasting is available at:

https://docs.ubports.com/en/latest/userguide/advanceduse/screencasting.html

Although I guess you'll have to adapt quite a bit.

capsia37 commented:
> Is it possible for you to record a video showing the effect before & after this change? Some hint for screencasting is available at:
> 
> https://docs.ubports.com/en/latest/userguide/advanceduse/screencasting.html
> 
> Although I guess you'll have to adapt quite a bit.

Hello @peat-psuwit ! I've tried a few times to capture it, but unfortunately it doesn't appear on the video, because it is too fast and the video doesn't have enough frames to capture it :(

Edit: I've recorded it on the desktop instead of using the phone:

With the flashing rectangle ( before ): https://streamable.com/bta5kb

And without the background blur to prevent flashing rectangle ( after ): https://streamable.com/zqimkr

capsia37 committed:
Fix infographics

- Don't show infographics if space is not enough

- Show infographics on cover for single user tablets

See:

https://github.com/ubports/unity8/issues/419#issuecomment-1013924494

https://github.com/ubports/unity8/issues/422

fredldotme commented:
I would very much like to see this included in time for OTA-22. It might be a small issue but it's a visually noticable difference, and makes something "amateur" look professional again.

It all depends on fixing the build-time issues. :)

capsia37 committed:
Fix #420

capsia37 commented:
> It all depends on fixing the build-time issues. :)

@fredldotme build fixed ;)

fredldotme commented:
Yeah, this looks really good. Thank you! :D

fredldotme reviewed (approved):
Like it, no complaints.

cibersheep commented:
I saw the changes in one of the latests dev: it feels sooo good, and I didn't know it.
Good work
